extensions/default/src/commandsModule.js

```diff
@@ -47,6 +47,19 @@ const commandsModule = ({ servicesManager, commandsManager }) => {
         title: 'DICOM Tag Browser',
       });
     },
+    toggleOverlays: () => {
```

Comment:
toggler for overlay

Reply:
who is running this? and why there is a notif shown

Reply:
removed the notification.
It was added during development. I thought it would be kind, but as this will toggle will make visual changes as well, I guess it was a duplicate message.